db/migrate/20150819150242_add_more_fields_to_project_details.rb

```diff
@@ -0,0 +1,133 @@
+class AddMoreFieldsToProjectDetails < ActiveRecord::Migration
+  def up
+    execute <<-SQL
+      drop function if exists public.published_states();
+      CREATE FUNCTION public.published_states() RETURNS text[]
+        LANGUAGE sql STABLE SECURITY DEFINER
+          AS $$
+            SELECT '{"online", "waiting_funds", "failed", "successful"}'::text[];
+          $$;
+
+      drop function if exists public.is_published(projects);
+      create function public.is_published(projects) returns boolean
+        language sql stable SECURITY DEFINER
+        as $$
+          select true where $1.state = ANY(public.published_states());
+        $$;
+
+      drop function if exists public.is_expired(projects);
+      CREATE FUNCTION public.is_expired(projects) RETURNS boolean
```

Comment:
We have a scope in the ruby model called ```expired```. We should rewrite that scope to use this function and avoid duplicate logic (besides, we can reuse the same specs in this way).